### PR TITLE
Added correct notation for big numbers on auction page

### PR DESCRIPTION
**Pull request by GeertWille**
### Currently:
Screenshot 2020-11-14 at 23 34 27

### After change:

Screenshot 2020-11-14 at 23 28 31

### CODE REVIEW

File: src/app/auction-page/auction-page.component.html [hunk 1]
```diff
@@ -111,7 +111,7 @@
         <span class="axn-form-block-pool-title">Auction price</span>
         <div class="axn-form-block-pool-axn">
           <span class="icon icon-flex icon__axion"
-            >{{ poolInfo.axnToEth }} AXN</span
+            >{{ poolInfo.axnToEth | bigNumberFormat: 0:true:false:18 }} AXN</span
```
Comment (Mvii7) [inline src/app/auction-page/auction-page.component.html:114]: Would this work?

{{ poolInfo.axnToEth | number }}

Comment (Mvii7) [reply src/app/auction-page/auction-page.component.html:114]: number:'1.0-2'

Comment (Mvii7) [reply src/app/auction-page/auction-page.component.html:114]: Looks like we are using our own pipe, I would like to know why

Comment (AudioSin) [reply src/app/auction-page/auction-page.component.html:114]: I just wanted to quickly fix this. Not create a new helper.

Comment (Mvii7) [reply src/app/auction-page/auction-page.component.html:114]: Sure, but the framework already has a number pipe, I see we have our own and we are using it everywhere instead. Ofc we are keeping it consistent, this looks ok

**Review by Mvii7 — APPROVED** [review]

**Review by Dellybro — APPROVED** [review]

**Review by Palmerg4 — APPROVED** [review]

**Pull request merged**